doc/GETTING_STARTED.md

```diff
@@ -29,87 +25,83 @@ The first step is to install libp2p in your project:
 npm install libp2p
 ```
 
-## Usage
-
-Configuring libp2p should not be a single step, but a continuous task. This includes configuring libp2p to achieve all the project requirements, as well as possible optimizations. Regardless of how you configure libp2p, the top level API will always remain the same.
-
-For creating a `js-libp2p` node you should use [Libp2p.create](./API.md#create). As stated in the API docs, we need to provide the [libp2p configuration](./CONFIGURATION.md) as a parameter. If you did not read the configuration document yet, it is worth you read it before continuing.
```

Comment:
I added this because I think we should guide people to use the API and CONFIGURATION docs, because as soon as they are experienced with `js-libp2p`, they should go through those docs instead

Reply:
Since the main readme is the landing page for most users I think it makes sense to have that distinction there. My concern with recommending people go read configuration first is that can be an overwhelming amount of information. We should help users get started quickly, and then send them to docs that will build on what they've learned here, and help them customize their nodes.

Reply:
I agree that it can be overwhelming to recommend people to read configuration first.
My idea here is to point them to subsections of the other documents, instead, of all document. So I agree with you on the configuration part, but I think we should keep the pointer do `./API.md#create`

Reply:
I added the link back to the first mention of `create`.